lib/nxt_vcr_harness.rb

```diff
@@ -66,5 +67,19 @@ def track_cassettes
     end
   end
 
-  module_function :enable_vcr_tag, :track_cassettes, :enable_vcr_cassette_helper, :track_cassettes_if
+  def strip_unneeded_headers_before_save
+    headers_to_strip = UnneededHeaders.default_headers_to_strip
+
+    yield headers_to_strip if block_given?
```

Comment:
Can we have a method documentation listing all the headers being stripped by default, so when you hover the method you can have the documentation on the IDE and know wich one to exclude from the list if you wanna keep.

Reply:
For something to be used often, sure! But this is something you only add to your repo once, so autocomplete isn't much use. Additionally, adding a manual list would double the size of the file, and get out of date easily.

As mentioned in the README, if you want to see the removed headers, you can just call `UnneededHeaders.default_headers_to_strip`